std/algorithm/searching.d

```diff
@@ -1792,6 +1796,7 @@ if (isRandomAccessRange!R1 && isBidirectionalRange!R2
     }
     // Stage 2: linear find
     size_t scout = needleLength - 1;
+outer:
```

Comment:
> Is that consensus?

No, just my opinion. I was never good with regex, so I don't know how many functions use your format, but I know there are four uses of the `label: loop` style in Phobos.